src/rdkafka_conf.c

```diff
@@ -403,8 +403,23 @@ static const struct rd_kafka_property rd_kafka_properties[] = {
           0, 1, 1 },
         { _RK_GLOBAL, "reconnect.backoff.jitter.ms", _RK_C_INT,
           _RK(reconnect_jitter_ms),
-          "Throttle broker reconnection attempts by this value +-50%.",
-          0, 60*60*1000, 500 },
+          "**Deprecated**: No longer used. See `reconnect.backoff.ms` and "
+          "`reconnect.backoff.max.ms`.",
+          0, 60*60*1000, 0 },
+        { _RK_GLOBAL, "reconnect.backoff.ms", _RK_C_INT,
+          _RK(reconnect_backoff_ms),
+          "The initial time to wait before reconnecting to a broker "
+          "after the connection has been closed. "
+          "The time is increased exponentially until "
+          "`reconnect.backoff.max.ms` is reached. "
+          "+-20% jitter is applied to each reconnect backoff. "
```

Comment:
I personally find this easier to read as one sentence. i.e. 

"The time is increased exponentially until reconnect.backoff.max.ms is reached +-20% jitter."

Reply:
That is more ambigious though, and possibly incorrect:
jitter is only applied to the individual backoff, not to the exponential increase.

Reply:
Fair point